forms/jqm/element.js

```diff
@@ -53,7 +53,10 @@ define(function (require) {
     },
 
     remove: function () {
-      this.$label = null;
+      if (this.$label) {
+        this.$label.remove();
+        this.$label = null;
```

Comment:
http://api.jquery.com/remove/

Description: Remove the set of matched elements from the DOM.

Similar to .empty(), the .remove() method takes elements out of the DOM. Use .remove() when you want to remove the element itself, as well as everything inside it. In addition to the elements themselves, all bound events and jQuery data associated with the elements are removed. To remove the elements without removing data and events, use .detach() instead.